apps/judicial-system/web/src/routes/Shared/CaseTable/CaseTable.tsx

```diff
@@ -207,7 +229,20 @@ const CaseTable: FC = () => {
       {user && // Wait until we have a user
         (table ? (
           <>
-            <SectionHeading title={table.title} />
+            <Box display="flex" alignItems="center">
```

Comment:
The check is visible on error. We should also hide it on error, right?
<img width="1447" alt="Screenshot 2025-05-28 at 11 31 15" src="https://github.com/user-attachments/assets/d2443e3e-d27a-4cbe-9dba-42dbea5feacd" />